unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -0,0 +1,674 @@
+          symbol_exprt generic_param{
+            "specalisedGeneric",
+            java_type_from_string("LDummyGeneric<Ljava/lang/Interger;>;")};
+          test_data.expected_params = {integer_param, ref_param, generic_param};
+          test_data.should_return_value = false;
+
+          validate_local_variable_lambda_assignment(
+            symbol_table,
+            instructions,
+            test_data,
+            std::regex(function_prefix_regex_str + R"(::\d+::paramLambda$)"));
+        }
+        THEN(
+          "The local variable should be assigned a non-null pointer to an "
+          "array parameter interface implementor")
+        {
+          lambda_assignment_test_datat test_data;
+          test_data.lambda_interface = "java::ArrayParameterLambda";
+          test_data.lambda_interface_method_descriptor =
+            ".Execute:([I[Ljava/lang/Object;[LDummyGeneric;)V";
+          test_data.lambda_function_id =
+            "java::LocalLambdas.lambda$test$2:"
+            "[I[Ljava/lang/Object;[LDummyGeneric;)V";
+
+          symbol_exprt integer_param{"primitive", java_type_from_string("[I")};
+          symbol_exprt ref_param{"reference",
+                                 java_type_from_string("[Ljava/lang/Object;")};
+          symbol_exprt generic_param{
+            "specalisedGeneric",
+            java_type_from_string("[LDummyGeneric<Ljava/lang/Interger;>;")};
+          test_data.expected_params = {integer_param, ref_param, generic_param};
+          test_data.should_return_value = false;
+
+          validate_local_variable_lambda_assignment(
+            symbol_table,
+            instructions,
+            test_data,
+            std::regex(
+              function_prefix_regex_str + R"(::\d+::arrayParamLambda$)"));
+        }
+        THEN(
+          "The local variable should be assigned a temp object implementing "
+          "ReturningLambdaPrimitive")
+        {
+          lambda_assignment_test_datat test_data;
+          test_data.lambda_interface = "java::ReturningLambdaPrimitive";
+          test_data.lambda_interface_method_descriptor = ".Execute:()I";
+          test_data.lambda_function_id = "java::LocalLambdas.lambda$test$3:()I";
+          test_data.expected_params = {};
+          test_data.should_return_value = true;
+          validate_local_variable_lambda_assignment(
+            symbol_table,
+            instructions,
+            test_data,
+            std::regex(
+              function_prefix_regex_str + R"(::\d+::returnPrimitiveLambda$)"));
+        }
+        THEN(
+          "The local variable should be assigned a temp object implementing "
+          "ReturningLambdaReference")
+        {
+          lambda_assignment_test_datat test_data;
+          test_data.lambda_interface = "java::ReturningLambdaReference";
+
+          test_data.lambda_interface_method_descriptor =
+            ".Execute:()Ljava/lang/Object;";
+
+          //"java::LocalLambdas.lambda$test$0:()V"
```

Comment:
How does this comment relate to the code here?

Reply:
Doesn't - removed